sycl/doc/OptionalDeviceFeatures.md

```diff
@@ -57,7 +51,7 @@ extension specification that enables indirect function calls.
 The term "exported device function" means a device function that is exported
 from a shared library as defined by [Device Code Dynamic Linking][5].
 
-[5]: <https://github.com/intel/llvm/blob/sycl/sycl/doc/SharedLibraries.md>
+[5]: <SharedLibraries.md>
```

Comment:
Technically it can be `[4]`.

Reply:
Fixed in 05c912cd2f60600b49117f0f9133ae249da12abf.

As this is only an editorial change, can this be merged without getting re-approvals?